From: Carlos Garnacho <carlosg@gnome.org>
Date: Sat, 12 Feb 2022 14:42:57 +0100
Subject: wacom: Allow NULL monitors in calibration

Unfortunately, this may be the case, it is better to show wonky UI
than no UI at all.

Origin: https://gitlab.gnome.org/GNOME/regolith-control-center/-/commit/3f12a7d
---
 panels/wacom/cc-wacom-page.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/panels/wacom/cc-wacom-page.c b/panels/wacom/cc-wacom-page.c
index b85bce7..b2f42ed 100644
--- a/panels/wacom/cc-wacom-page.c
+++ b/panels/wacom/cc-wacom-page.c
@@ -313,8 +313,7 @@ calibrate (CcWacomPage *page)
 		/* The display the tablet should be mapped to could not be located.
 		 * This shouldn't happen if the EDID data is good...
 		 */
-		g_critical("Output associated with the tablet is not connected. Unable to calibrate.");
-		return;
+		g_critical("Output associated with the tablet is not connected. Calibration may appear in wrong monitor.");
 	}
 
 	old_calibration = g_settings_get_value (page->wacom_settings, "area");
